# This message is used to assign values to all servos on an expander board at once.
float32 servo0
float32 servo1
float32 servo2
float32 servo3
float32 servo4
float32 servo5
float32 servo6
float32 servo7
float32 servo8
float32 servo9
float32 servo10
float32 servo11
float32 servo12
float32 servo13
float32 servo14
float32 servo15
